lib/components/Navigation/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { forwardRef } from "react"
+import {
+  Breadcrumb as BreadcrumbComponent,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbPage,
+  BreadcrumbSeparator,
+} from "../../../ui/breadcrumb"
+
+interface BreadcrumbsType {
+  icon: IconType
+  routes: RouteType[]
+  title: string
+}
+
+interface RouteType {
+  title: string
+  url: string
+}
+
+export const Breadcrumb = forwardRef<HTMLDivElement, BreadcrumbsType>(
+  ({ icon, routes, title }, ref) => {
+    return (
+      <BreadcrumbComponent
+        className="flex min-h-6 flex-row border-x-0 border-t-0 border-dashed border-b-muted px-5 py-3"
```

Comment:
I have the feeling that we should bake in all these classes inside ShadCN's component, right? What I usually try to do is to change the original component's API as little as possible, but change its aspect if needed.

This whole "trying to keep ShadCN alive" is quite messy though, so both options are good if the public API stays stable.

Reply:
Yeah, I don't think there's a right answer, end of day what we are exposing is the same, regardless of where we put the styles. I can't think of any major pros / cons of either option.